incubator/appdaemon/0.1.0/test_values.yaml

```diff
@@ -0,0 +1,52 @@
+image:
+  repository: ghcr.io/truecharts/appdaemon
+  pullPolicy: IfNotPresent
+  tag: "v4.0.8"
+
+strategy:
+  type: Recreate
+
+# Configure the Security Context for the Pod
+podSecurityContext:
+  runAsNonRoot: false
+  runAsUser: 0
+  runAsGroup: 0
+  fsGroup: 0
+
+
+env:
+  HA_URL: "http://192.168.5.31:8123"
```

Comment:
Uhmm... Be aware: When doing tests there is NO Home Assistant instance at all!
Also, you should use the service name, not the IP.